srml/assets/src/lib.rs

```diff
@@ -0,0 +1,228 @@
+// Copyright 2017-2018 Parity Technologies (UK) Ltd.
```

Comment:
Nitpick, you started this probably 2018^^

Reply:
i think it's fine to keep copyright/license headers consistent throughout